FirebasePhoneAuthUI/CountryCode/FUICountryCodes.m

```diff
@@ -163,19 +154,58 @@ - (FUICountryCodeInfo *)countryCodeInfoForPhoneNumber:(NSString *)phoneNumber {
   return nil;
 }
 
-- (FUICountryCodes *)searchCountriesByName:(NSString *)countryName {
-  if ([countryName length] == 0) {
-    return nil;
-  } else {
-    NSArray *results =
-        [self.countryCodesArray filteredArrayUsingPredicate:
-            [NSPredicate predicateWithFormat:FUI_JSON_COUNTRY_NAME_PREDICATE, countryName]];
-    return [[FUICountryCodes alloc] initWithCountriesArray:results];
+- (void)blackListCountries:(NSSet<NSString *> *)countries {
+  NSMutableArray<NSDictionary *> *array =
+      [[NSMutableArray alloc] initWithCapacity:_countryCodesArray.count];
```

Comment:
This should be `countries.count`, since the loop below will call `addObject` at most `countries.count` times.

Reply:
Done

Reply:
Oh this one should be _countryCodeArry.count. 
For instance if I have 100 element in the countryCodeArray and 1 in the blacklist set, I should have 99 elements in the result. 
I changed the whitelist one to countries.count.